study.md

```diff
@@ -34,7 +34,7 @@ including class. Why might Array redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+They share the methods "drop" and "include?".  I'm honestly not sure what the second part of the question is asking.  All of the methods that are shared that I could find aren't 'redefined', they have the same definition under an array or an enumerable.
```

Comment:
`Array` might redefine methods included from `Enumerable` if custom behavior is desired.  For example, `Array` might redefine `each` to only return elements with even indices on each iteration.  This is useful if you're writing a custom collection class and `each` needs to traverse an array in a different way.